docusaurus/docs/cloud/account/settings.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Account Settings
+displayed_sidebar: cloudSidebar
+description: View and manage your projects on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/account/settings.html
+---
+
+# Account Settings
+
+The **Account Settings** page enables you to manage your account details and preferences. There are four sections available: 
+
+## General
+
+The **General** tab enables you to edit the following details for your account:
+
+* **Name**: The name associated with your account.
+* **Email**: The email address associated with your account.
+* **Delete account**: This will ***permanently and irreversibly*** delete your account and all its associated data.
+
+## Preferences
+
+The **Preferences** tab enables you to edit the following details for your account:
+
+* **Notifications**: Enable or disable email notifications for project activity.
```

Comment:
Same, `Notification settings` won't be there for SL; please remove lines 24 & 25.